240 convert event model to use dls python skeleton (#241)
bluesky/event-model: docs/developer/tutorials/dev-install.rst

Developer install
=================

These instructions will take you through the minimal steps required to get a dev
environment setup, so you can run the tests locally.

Clone the repository
--------------------

First clone the repository locally using `Git
<https://git-scm.com/downloads>`_::

    $ git clone git://github.com/bluesky/event-model.git

Install dependencies
--------------------

> You can choose to either develop on the host machine using a `venv` (which
> requires python 3.8 or later) or to run in a container under `VSCode
> <https://code.visualstudio.com/>`_
> 
> .. tab-set::
> 
>     .. tab-item:: Local virtualenv
> 
>         .. code::
> 
>             $ cd event-model
>             $ python3 -m venv venv
>             $ source venv/bin/activate
>             $ pip install -e '.[dev]'
> 
>     .. tab-item:: VSCode devcontainer
> 
>         .. code::
> 
>             $ vscode event-model
>             # Click on 'Reopen in Container' when prompted
>             # Open a new terminal

Reviewer: ```suggestion
You should install into a `venv` (which requires python 3.8 or later):

.. code::

    $ cd event-model
    $ python3 -m venv venv
    $ source venv/bin/activate
    $ pip install -e '.[dev]'
```
